pkg/controller/credentialsrequest/actuator/actuator.go

```diff
@@ -32,4 +32,8 @@ type Actuator interface {
 	Update(context.Context, *minterv1.CredentialsRequest) error
 	// Checks if the credentials currently exists.
 	Exists(context.Context, *minterv1.CredentialsRequest) (bool, error)
+	// NeedsUpdate will answer the question whether there is a need to update credentials
+	// based on whether any creds exist, or whether the existing creds no longer
+	// satisfy the CredentialsRequest
+	NeedsUpdate(context.Context, *minterv1.CredentialsRequest) (bool, error)
```

Comment:
If this is working we can probably go with it for now. 

Ultimately I'm worried this shouldn't be done in the controller and actuator interface, I don't think it's a given that the next cloud we implement will want to follow the same paradigm with passthrough creds, and it's probably better handled within the actuator itself.

However we don't have time right now, so if this is working we can go with it and investigate later.